tests/unit/vue-ld.spec.js

```diff
@@ -95,4 +95,27 @@ describe('VueLd Plugin', () => {
     expect(vueLdCallback).toBeCalled();
     expect(vueLdCallback.mock.instances[0]).toBe(wrapper.vm.$ld);
   });
+
+  it('stubs flags when passed the option', async () => {
+    localVue.use(VueLd, {
+      ...vueLdOptions,
+      flagsStub: new Proxy(
+        {
+          never: false,
+        },
+        {
+          get(obj, prop) {
+            const value = obj[prop];
+            return value === undefined ? true : value;
+          },
```

Comment:
Given this will probably be used as a reference implementation, it'd be helpful to include a test showing how adding/changing flags is supported by the stub. (Or provide an example in the readme.)

Reply:
Does my change to the test reflect what you were asking for?